monai/engines/trainer.py

```diff
@@ -57,8 +58,27 @@ def run(self) -> None:
         self.scaler = torch.cuda.amp.GradScaler() if self.amp else None
         super().run()
 
-    def get_train_stats(self) -> dict[str, float]:
-        return {"total_epochs": self.state.max_epochs, "total_iterations": self.state.epoch_length}
+    def get_train_stats(self, *others) -> dict[str, float]:
```

Comment:
Unless I'm out of the loop on why, I would call both methods just `get_stats` so that code can be written to use either class.

Reply:
Updated in the latest commit.

Thanks.